app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -33,19 +32,15 @@ function initializeDrawerSliders() {
 }
 
 function listenForNarrowMenuClick(event) {
-  const navLinks = document.getElementsByClassName('narrow-nav-menu');
+  const navLinks = document.getElementById('narrow-nav-menu');
```

Comment:
Since there is only one `'narrow-nav-menu'` element on the page, you might as well rename the variable accordingly.

```suggestion
  const navLink = document.getElementById('narrow-nav-menu');
```